src/runtime_src/driver/zynq/drm/zocl/sched_exec.h

```diff
@@ -69,13 +69,19 @@ enum cmd_state {
  * @OP_START_CU:       start a workgroup on a CU
  * @OP_START_KERNEL:   currently aliased to ERT_START_CU
  * @OP_CONFIGURE:      configure command scheduler
+ * @OP_CONFIG_SKERNEL: configure soft kernel
+ * @OP_START_SKERNEL:  start soft kernel
+ * @OP_SK_UNCONFIG:    unconfigure a soft kernel
  */
 enum cmd_opcode {
```

Comment:
Can these not come directly from ert.h?  Redefining the constants is not cool, renaming them is fine, but replace the scalar values with the symbolic enum values from ert.h.

Reply:
I totally agree with you on this. However, this is a legacy behavior that zocl drivers itself copy (redefine and rename) what have been defined in ert.h. This PR just follows what is done for hard kernel (OP_START_KERNEL, OP_CONFIGURE, etc.).
Removing overlaping of sched_exec.h and ert.h will be addressed in a different PR.